pandas/tseries/offsets.py

```diff
@@ -527,23 +543,10 @@ def _from_name(cls, suffix=None):
 class BusinessMixin(object):
     """ mixin to business types to provide related functions """
 
-    # TODO: Combine this with DateOffset by defining a whitelisted set of
-    # attributes on each object rather than the existing behavior of iterating
-    # over internal ``__dict__``
-    def __repr__(self):
-        className = getattr(self, '_outputName', self.__class__.__name__)
-
-        plural = ''
-        if abs(self.n) != 1:
-            plural = 's'
-
-        n_str = ''
-        if self.n != 1:
-            n_str = '{n} * '.format(n=self.n)
-
-        repr_content = ''.join([n_str, className, plural, self._repr_attrs()])
-        out = '<{content}>'.format(content=repr_content)
-        return out
+    @property
+    def offset(self):
+        # Alias for backward compat
```

Comment:
better comment on what this attribute is (its not for backward compat, rather its the API).

Reply:
Not sure I understand.  It explicitly is for backward compat since we are trying to standardize on `_offset`.

Reply:
``offset``is a user API, the ``_offset`` is merely an implementation detail (e.g. how we implement it). add a doc-string on what this returns.

Reply:
OK